libunwind/src/DwarfInstructions.hpp

```diff
@@ -21,6 +21,9 @@
 #include "DwarfParser.hpp"
 #include "config.h"
 
+#ifdef __CHERI_PURE_CAPABILITY__
+#include <cheri/cheric.h>
```

Comment:
We just use the __builtin_cheri_foo directly; so far libunwind doesn't use any CHERI headers. Definitely not cheri/cheric.h though, that's CheriBSD-specific.

Reply:
What should I include to use the `CHERI_PERM_EXECUTIVE` below?

Reply:
There should be an equivalent macro pre-defined by the compiler itself, something like `__ARM_CAP_PERM_EXECUTIVE__`